setup.py

```diff
@@ -152,6 +155,7 @@ def run(self):
     "filelock",
     "black==19.10b0",
     "wsgidav",
+    "osfclient==0.0.4",
```

Comment:
So this will require the user to `pip install osfclient` themselves? Or can it be an optional dependency when you `pip install dvc[osf]`?

Reply:
p.s. asking per iterative/dvc.org/pull/1958

Reply:
@jorgeorpinel, this is only for the tests. There's `osf` above already.

Reply:
OK. My question is in general about the package though, not this specific line. Thanks